### ISSUE TITLE
[Accessibility][JAWS][Bulk Edit][edit student accomodations]: The phrase “1 of 1 page” is announced incorrectly and in parts by screen readers.

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#2200]
[A11Y-4170](https://jira.cengage.com/browse/A11Y-4170)

Platform:
Windows 11 / Chrome /Keyboard
Mac15/ Safari/ Voiceover

Steps to reproduce:
1. Open the URL "https://faculty-stage.cengage.com/"
2. Click on "sign in" button
3. On sign in page enter instructor email and password and do sign in
4. Select "My course" from main menu
5. Select course key: MTPQGMDZD55G
6. Navigate near "Bulk Edit" and activate it.
7. Select the content and move near "Edit X activities" button.
8. Expand the button, and select "edit student accomodations" and expand it.
9. Navigate to the pagination control or content area where “1 of 1 page” is visually displayed.
10. Observe that the phrase “1 of 1 page” is announced incorrectly and in parts by screen readers

Actual Result:
The screen reader announces “1” … pause … “of” … pause … “1” … pause … “page” as separate, disconnected pieces, breaking the logical meaning of the full phrase.

Expected Result:
The full phrase “1 of 1 page” should be read as a single, continuous, and coherent sentence by the screen reader without unnecessary pauses or disconnection.

WCAG Guideline:
1.3.1: Info and Relationships

User Impact:
Screen reader users may become confused or misinterpret the meaning of pagination status. This impacts users' understanding of where they are in a sequence of pages or items and may hinder efficient navigation.

Recommendation:
Wrap the entire phrase “1 of 1 page” within a single semantic element (e.g., a `<span>` or `<div>`), avoiding splitting the phrase across multiple tags unless properly associated.

Screencast:
https://app.screencast.com/tR590pnBKpIKV

**Comment by silvalaura** [comment]
@olehnoskov-cengage is this different to https://github.com/cengage/react-magma/pull/2037?

**Comment by olehnoskov-cengage** [comment]
> [@olehnoskov-cengage](https://github.com/olehnoskov-cengage) is this different to [#2037](https://github.com/cengage/react-magma/pull/2037)?

Yes, it is.

**Comment by silvalaura** [comment]
Looks good to me on VoiceOver (fixed indirectly). @nikitaorliak-cengage to check NVDA, @moathabuhamad-cengage to check JAWS.

**Comment by nikitaorliak-cengage** [comment]
Related to the `Table` component, not `Pagination`. Probably will be fixed in this PR: https://github.com/cengage/react-magma/pull/2194

**Comment by nikitaorliak-cengage** [comment]
Tested on Windows + NVDA and MacOS + VO, everything works as expected.
